### DISCUSSION TITLE
[Question][Hw3] Make on test directory failed with Out Of Memory.

### DISCUSSION THREAD

**Discussion by Leechangjun1011** [prosyslab-classroom/cs348-information-security#541] [Q&A]
Make on test directory is  failed with oom as following capture.
Same error is occurred when I try Make after make clean.
I run it on given  server.
How can I  solve this  problem?
![makeoom](https://user-images.githubusercontent.com/62602090/117017522-9e3f9500-ad2e-11eb-8b61-83f54b332062.JPG)

**Comment by KihongHeo** [comment]
I don't think `make` causes OOM.
Probably your sanitizer does which is called in the Makefile.

To make it sure, please run your sanitizer without using make.

**Comment by Leechangjun1011** [comment]
Thanks for help.
